packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift

```diff
@@ -2,13 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import FlutterMacOS
 import Foundation
 
+#if os(iOS)
+import Flutter
+#elseif os(macOS)
+import FlutterMacOS
+#endif
```

Comment:
This would be avoided by https://github.com/flutter/flutter/issues/70413

Reply:
Yep. This is something I could try to make time for in 2023 maybe?

Reply:
It would be really great to unify these frameworks and APIs more, as you know 🙂